Library/Formula/rtmidi.rb

```diff
@@ -1,23 +1,17 @@
 class Rtmidi < Formula
   desc "C++ classes that provide a common API for realtime MIDI input/output"
   homepage "https://www.music.mcgill.ca/~gary/rtmidi/"
-  url "https://github.com/thestk/rtmidi/archive/2.1.0.tar.gz"
-  sha256 "52e6822fc413d5d3963c5b7bfe412ed69233bb81a7f04d6097f5b56aafa28934"
-
-  bottle do
-    cellar :any
-    sha256 "2abe3e651405114e4a667eb1d3491be8fde15a62d7605622449a92a6ab78a443" => :yosemite
-    sha256 "4516090de14fa3f4caa5e7bc277f4d64d269dcad32488cc195395572803e6186" => :mavericks
-    sha256 "329d88e5bbe8d96549707288b9599213cc16f9df64880d6de035fc0f50c10d14" => :mountain_lion
-  end
+  url "https://github.com/thestk/rtmidi/archive/2.1.1.tar.gz"
+  sha256 "a015fbed67c777c7bc7bbcb96c07386ba3a8ff21006b411331e6f0b9f3f14d11"
 
   depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
 
   def install
-    system "autoconf"
-    system "./configure", "--prefix=#{prefix}"
-    system "make", "librtmidi.a"
-    lib.install Dir["*.a", "*.dylib"]
+    system "./autogen.sh"
```

Comment:
If it's not being used, don't worry about it. There's no `make install` step to correlate to passing a prefix so not having to use one here wouldn't be _that_ unusual. Possibly just worth peeking into the `autogen` script to see if there's a `prefix` argument there.